recipes/pasta/1.7.8/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+work_dir=$PREFIX/pasta-build
+install_dir=$PREFIX/bin
+
+mkdir -p $work_dir
+mkdir -p $work_dir/pasta
+
+cp -R ./* $work_dir/pasta
+cd $work_dir
+
+unamestr=`uname`
+if [ $unamestr == 'Linux' ];
+then
+    wget --no-check-certificate https://github.com/smirarab/sate-tools-linux/archive/a3e6f56.tar.gz
+    tar -xzf a3e6f56.tar.gz
+    mv sate-tools-linux-a3e6f56372599ffacf1bd43adb2d67cf361228e2 sate-tools-linux
+    cd pasta
+    $PYTHON setup.py install
+    mkdir $install_dir/sate-tools-linux
+    cp -R $work_dir/sate-tools-linux/* $install_dir/sate-tools-linux/
+    # Handle a PASTA bug.
+    cp $install_dir/sate-tools-linux/hmmalign $install_dir/sate-tools-linux/hmmeralign
```

Comment:
Are these the only tools you fetch with wget? maybe you can just depend on hmmer?